$NetBSD: patch-configure,v 1.2 2012/02/15 23:13:32 hans Exp $

Fix configury with libaudiofile-0.3+.
Fix interpreter in get-wsound-flags.

--- configure.orig	2011-12-19 13:10:25.000000000 +0000
+++ configure
@@ -3252,11 +3252,11 @@ echo "configure:3248: checking for AUDIO
     AUDIOFILE_LIBS=`$AUDIOFILE_CONFIG $audiofileconf_args --libs`
     AUDIOFILE_CFLAGS=`$AUDIOFILE_CONFIG $audiofileconf_args --cflags`
     audiofile_major_version=`$AUDIOFILE_CONFIG $audiofile_args --version | \
-           sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\1/'`
+           sed 's/\([0-9]*\).\([0-9]\)\([0-9]*\)/\1/'`
     audiofile_minor_version=`$AUDIOFILE_CONFIG $audiofile_args --version | \
-           sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\2/'`
+           sed 's/\([0-9]*\).\([0-9]\)\([0-9]*\)/\2/'`
     audiofile_micro_version=`$AUDIOFILE_CONFIG $audiofile_config_args --version | \
-           sed 's/\([0-9]*\).\([0-9]*\).\([0-9]*\)/\3/'`
+           sed 's/\([0-9]*\).\([0-9]\)\([0-9]*\)/\3/'`
     if test "x$enable_audiofiletest" = "xyes" ; then
       ac_save_CFLAGS="$CFLAGS"
       ac_save_LIBS="$LIBS"
@@ -4272,7 +4272,7 @@ test "$no_create" = yes || ${CONFIG_SHEL
 
 
 cat <<EOF >get-wsound-flags
-#!/bin/sh
+#!${TOOLS_PATH_SH}
 
 prefix="$prefix"
 exec_prefix=\$prefix
